### ISSUE TITLE
 The deployment of functions failed with the following errors: javy_quickjs_provider_v1::invoke

### ISSUE THREAD

**Issue by namanh0705** [Shopify/function-examples#278]
Hi everyone, today i deploy my function but got this error, so i remove my app and generate newest extension again, but still get error when deploy to shopify
 Can anyone help me? thanks

 The deployment of functions failed with the following errors:                                                
[                                                                                                          
{                                                                                              
"field": null,                                                                                           
"message": "Wasm validation failed: unknown import: `javy_quickjs_provider_v1::invoke` has not been
defined",                                                                                                    
"tag": "user_error"                                                                                      
 } 
]

**Comment by SCDevSupport** [comment]
fasf

**Comment by namanhhex** [comment]
> +1, getting this error as well.
> 
> EDIT: No longer getting this error. `npm run deploy` runs successfully for me.

Thanks friend, let me try again

**Comment by nickwesselman** [comment]
This issue with javy has been resolved. Shopify CLI should pull the latest automatically.
